acid/features/history/tests/test_model.py

```diff
@@ -12,49 +12,124 @@
 from ..model import ZuulBuildSet
 
 
-@db_session
 @pytest.mark.unit
 class TestZuulBuildSet(DatabaseTestCase):
     # (kam193) TODO: randomize times
```

Comment:
Remove this comment. It is obsolete as we added explicit time declaration for builds.